docs/src/styles/docs/main.scss

```diff
@@ -19,16 +19,17 @@
 
 .docs-content-body {
   padding: var(--amplify-space-xxl);
-  
-  & table {
+
+  & table:not(.amplify-table) {
```

Comment:
It seems we can get rid of `&` since they are nested. It should be a default behavior